### DISCUSSION TITLE
Welcome to cmf Discussions!

### DISCUSSION THREAD

**Discussion by annmary-roy** [HewlettPackard/cmf#157] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by kkm-horikawa** [comment]
## 決定 ✅

名称を正式決定しました：

| 項目 | 名称 |
|------|------|
| 会社/ブランド | **Serviio** |
| プロダクト | **Ordio** |
| パッケージ | `com.serviio.ordio` |
| リポジトリ | `ordio` |

PR #86 で実装済み。
